opentelemetry-collector.spec.in

```diff
@@ -85,13 +84,15 @@ fi
 %files
 %{_unitdir}/%{name}.service
 %{_sysconfdir}/opentelemetry-collector/configs/00-default-receivers.yaml
-%{_sysconfdir}/opentelemetry-collector/configs/00-microshift-receivers.yaml
 
 %license %{golicenses}
 %doc %{godocs}
 %{_bindir}/*
 
 %changelog
+* Fri June 28 2024 Benedikt Bongartz <bongartz@redhat.com> - 0.102.0
```

Comment:
The Makefile specifies OTELCOL_VERSION as 0.102.1
I guess this was still 0.102.0 on June 28. Can you version bump the changelog entry?

Reply:
```suggestion
* Fri June 28 2024 Benedikt Bongartz <bongartz@redhat.com> - 0.102.1
```